packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -632,7 +632,7 @@ class LineItem extends StatefulWidget {
     this.activeSearchMatch,
   }) : super(key: key);
 
-  static const _hoverDelay = Duration(milliseconds: 50);
+  static const _hoverDelay = Duration(milliseconds: 150);
```

Comment:
why is this better?

Reply:
We don't want to show the hover card too quickly. This _feels_ about right.